File: Models/OrderProductModel.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace EvangelionERP.Models
{
    [Table(name: "tab_orders_itens")]
    public class OrderProductModel
    {
        [Key]
        public int Cod { get; set; }
        public virtual OrderModel Order { get; set; }
        public int OrderCod { get; set; }
        public virtual ProductModel Product { get; set; }
        public int ProductCod { get; set; }   
        public decimal Quantity { get; set; }   
        public decimal Price { get; set; }   
        public string Name { get; set; }
        public bool? FlOutput { get; set; }
    }
}

File: CrudAngular/frontend/src/app/componentes/product/orderProduct.model.ts
import { FormControl, FormGroup } from "@angular/forms";

export class OrderProduct {
    cod: number = 0;
    orderCod: number = 0;
    name: string = "";
    quantity: number = 0;
    price: number = 0;
    productCod: number = 0;
    flOutput: boolean = true;

    static asFormGroup(product: OrderProduct): FormGroup {
        const fg = new FormGroup({
          cod: new FormControl(product.cod),
          name: new FormControl(product.name),
          quantity: new FormControl(product.quantity),
          price: new FormControl(product.price)
        });
        return fg;
    }

}